src/main/kotlin/org/amshove/kluent/CharSequenceBacktick.kt

```diff
@@ -24,9 +24,9 @@ fun CharSequence.`should be blank`() = this.shouldBeBlank()
 
 fun CharSequence?.`should be null or blank`() = this.shouldBeNullOrBlank()
 
-infix fun String.`should equal to`(theOther: String) = this.shouldEqualTo(theOther)
+infix fun String.`should be equal to`(theOther: String) = this.shouldEqualTo(theOther)
```

Comment:
Since this is a breaking change, I suggest deprecating all occurences of `should equal to` and removing them in Kluent 2.x

Reply:
Ok. Good call. I'll make those changes and update the PR :)